Resources/Prototypes/_Goobstation/Entities/Clothing/OuterClothing/specific.yml

```
@@ -21,8 +21,9 @@
     walkModifier: 0.9
     sprintModifier: 0.9
   - type: HeldSpeedModifier
-  - type: ToggleableClothing
-    clothingPrototype: ClothingHeadHelmetHEV
+  - type: ToggleableClothing # Goobstation - Modsuits change
+    clothingPrototypes: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)